Team_47/QuizCreator/WebContent/instruction.jsp

```diff
@@ -24,65 +17,44 @@
 		mulbutton = "button";
 	}
 	String title = "The maximum time for quiz is 30 minutes.  No partial grading is avaialable";
-
 %>
 <head>
 <meta charset="UTF-8">
 <title>Instruction</title>
 </head>
 <body style="background-color:#FFF8DC">
 <div class = "content">
-<%@ page import="org.team47database.*"%>
-    <%@ page import="java.util.List"%>
-
-
-	
 	<%
 	String quizId=request.getParameter("quizId");
-Object quizid =  request.getSession().getAttribute("quizid");	
+	Object quizid =  request.getSession().getAttribute("quizid");	
 	String quizid1= String.valueOf(quizid) ;  
-	
-  	//Object enrolled_id =  request.getSession().getAttribute("quizId");	
-	//String enrolled_id1  =  String.valueOf(enrolled_id);
 	String HTMLContent = "";
 	String content="";
  	StringBuilder buf = new StringBuilder();
  	
  	Quiz quiz=new QuizDAOImpl().getQuiz(Integer.valueOf(quizId));
  	buf.append(String.format( 	
  			"<div class=\"question-top-area\">"+
- 		 			"<div class=\"question-title\""+
-
- 							"<h2>Quiz Title:%s </h2>"+
- 							"<hr>"+
- 							"<h3>Quiz Points:%s </h3>"+
- 		 					"<h3>Quiz Group:%s </h3>"+
-					"</div>"+
-			"</div>"	+
- 	"<div top=100px; class =\"instruction_area\">"+
-	"<p     style=\"margin-top: 160px;\">Quiz Instructions</p>"+
-	"<input id = \"instruction\" type = \"text\" value=\"1) %s\"style=\"height:50px;width:600px;\" disabled=\"disabled\">"+
-	"</input>"+
-	"</div>"+
-     "<div class=\"bottom\">"+
-
-"</div>",quiz.getTitle(),
-	String.valueOf(quiz.getTotal_points()),
-	 	quiz.getQuiz_group(),
-
-	quiz.getInstructions()
-
+ 		 	"<div class=\"question-title\""+
+ 			"<h2>Quiz Title: %s </h2>"+
+ 			"<hr>"+
+ 			"<h3>Quiz Points: %s </h3>"+
+ 		 	"<h3>Quiz Group: %s </h3>"+
+			"</div>"+
+			"</div>"+
+ 			"<div top=100px; class =\"instruction_area\">"+
+			"<p style=\"margin-top: 160px;\">Quiz Instructions</p>"+
+			"<input id = \"instruction\" type = \"text\" value=\"1) %s\"style=\"height:50px;width:600px;\" disabled=\"disabled\">"+
```

Comment:
This line is wrapping on my diff view, but given that it is wrapping, my guess is the line is too long

Reply:
I guess they are not a problem. This file is originally not mine and I only styled them. I have run it and look the same as before. Or if u want me to separate line 47 more?

Reply:
I suggest breaking up the string.